sensu/plugins/check-cinder-sessions.py

```diff
@@ -56,7 +56,14 @@ def _get_local_active_instance_volumes():
 
 def _get_active_iscsi_volumes():
     volumes = set()
-    out = subprocess.check_output(['iscsiadm', '-m', 'session'])
+    try:
+        out = subprocess.check_output(['iscsiadm', '-m', 'session'])
+    except Exception, e:
+        # iscsiadm returns 21 if there are no active sessions.
+        if e.returncode == 21:
+            return list(volumes)
```

Comment:
this reads strangely.  This becomes much cleaner if you move the iteration over "out" lines into the try/except block. Then all you need is:
if e.returncode == 21:
    pass